docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -97,7 +97,7 @@ To list and search only stable releases:
   - At nuget.org, deselect the **Include prerelease** checkbox in the advanced search panel.
   - In the Visual Studio NuGet Package Manager UI, deselect the **Include prerelease** checkbox next to the Search box.
 
-The Visual Studio Package Manager Console, NuGet CLI, and dotnet CLI tools don't include prerelease versions by default. To include prerelease versions:
+The Visual Studio Package Manager Console, NuGet CLI, and dotnet CLI tools doesn't include prerelease versions by default. To include prerelease versions:
```

Comment:
FWIW, "don't" feels better than "doesn't" to me. I don't know theoretical English grammar rules sufficiently well (despite being a native English speaker) to know if either is technically correct, but I think most people I would talk to would use "don't".

Reply:
> FWIW, "don't" feels better than "doesn't" to me. I don't know theoretical English grammar rules sufficiently well (despite being a native English speaker) to know if either is technically correct, but I think most people I would talk to would use "don't". edit: while Word doesn't complain about grammar in this sentence with either "doesn't" or "don't", Grammarly suggests using "don't".

Well if we go by grammar then always in case of present tenses singular helping verb (in this case helping verb is DOESN'T)  is used with plural subjects (in this case subject is TOOLS). And if the Word is not throwing any sort of error then it is completely on the speaker to go by verbs of their choice , I would prefer grammar over what suits better while speaking. If you want me to change it back as it was previously I can do that I have no problem. :)

Reply:
> I would prefer grammar over what suits better while speaking.

I found the Microsoft docs guidelines: https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#write-like-you-speak

Therefore, your proposal contradicts the style guide for docs.